rust/Makefile

```diff
@@ -147,9 +148,17 @@ $(objtree)/rust/alloc.o: $$(RUST_LIB_SRC)/alloc/src/lib.rs \
     $(objtree)/rust/compiler_builtins.o FORCE
 	$(call if_changed_dep,rustc_library)
 
+# link_time_panic.o is deliberately only compiled as a dependency but not
+# included in obj-y.
```

Comment:
It does look like the `.o` is linked into `kernel.o`, is the idea that it's linked there, but not into the final `.ko`?

Reply:
`kernel.o` has `link_time_panic.o` as a Make dependency, but it will not link against it.